src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/VerseCard.kt

```diff
@@ -0,0 +1,71 @@
+package org.wycliffeassociates.otter.jvm.app.widgets
+
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.geometry.Pos
+import javafx.scene.layout.VBox
+import javafx.scene.paint.Color
+import org.wycliffeassociates.otter.jvm.app.ui.chapterPage.model.Verse
+import tornadofx.*
+import tornadofx.Stylesheet.Companion.root
+
+class VerseCard(verse: Verse, color: String, icon: MaterialIcon, text: String, recordVerseCard: Boolean) : VBox() {
+
+    init {
+        with(root) {
+            when (!recordVerseCard) { //used for takes and edit contexts
```

Comment:
I think idiomatic Kotlin prefers `if`-`else` over `when` if only checking two possibilities

Reply:
yes. looks like when is supposed to be a replacement for switch cases